hooks/useSiteMetadata.ts

```diff
@@ -0,0 +1,21 @@
+// import { useStaticQuery, graphql } from 'gatsby';
+
+// /**
+//  * @returns Site metadata from gatsby-config.ts
+//  */
+// export const useSiteMetadata = () => {
+//   const { site } = useStaticQuery(
+//     graphql`
+//       query SiteMetadataQuery {
+//         site {
+//           siteMetadata {
+//             title
+//             description
+//             siteUrl
+//           }
+//         }
+//       }
+//     `,
+//   );
+//   return site.siteMetadata;
+// };
```

Comment:
This is actually being removed completely in #97 . There's a completely new way to deal with html head metadata stuff in nextjs v13 app directory approach